modules/ROOT/pages/am-permissions.adoc

```diff
@@ -3,15 +3,20 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The *Access Management* section of Anypoint Platform enables you (as an Organization Administrator) to control user access to dashboards, reports, and metrics by providing two predefined product-specific permissions:
+Anypoint Monitoring has two product-specific permissions that control access to assets within Anypoint Monitoring:
 
-* Anypoint Monitoring Administrator. Allows users to view, create, modify, and delete all content in Anypoint Monitoring.
-+
-* Anypoint Monitoring Viewer. Allows users to view but not modify content in Anypoint Monitoring.
+Anypoint Monitoring Administrator:: Allows users to view, create, modify, and delete all content in Anypoint Monitoring.
+Anypoint Monitoring Viewer:: Allows users to view but not modify content in Anypoint Monitoring.
+
+The content you can access when you have the Anypoint Monitoring Administrator and Anypoint Monitoring Viewer permissions depends on other asset-specific permissions.
+To view or modify content related to specific apps and APIs, you also need the appropriate access permissions for those assets.
+For example, a user who has the Anypoint Monitoring Viewer permission who wants to view Anypoint Monitoring data for an on-premises app also needs the Read Applications permission in the same environment as the app.
```

Comment:
"Read Applications" -> "Runtime Manager Read Applications"? 
Would it be a good idea to keep the pattern `{product namespace} {permission name}`?

Reply:
We should explicitly specify the permissions that are required for monitoring data e.g. read applications.

Reply:
I hesitate to do that since these permissions models may change. There are also lots of variables for which permissions are needed based on what the users need to see.

Reply:
```suggestion
For example, a user who has the Anypoint Monitoring Viewer permission and wants to view Anypoint Monitoring data for an on-premises app also needs the Read Applications permission in the same environment that the app is deployed to.
```